app/dns/dns.go

```diff
@@ -9,6 +9,7 @@ import (
 	"strings"
 	"sync"
 
+	dns_lib "github.com/miekg/dns"
```

Comment:
Import this lib add quite a bit to the binary size. Can we hard code a constant for now?